### ISSUE TITLE
Unhandled error: Unable to make 1951.0 MB space on cpu. 

### ISSUE THREAD

**Issue by sourceful-karlson** [brycedrennan/imaginAIry#345]
![Screenshot 2023-06-05 at 12 40 41](https://github.com/brycedrennan/imaginAIry/assets/66862653/91b63a74-b0d1-4f3e-9d3c-eeba61abba86)

I encountered the above issue, what would be a quick fix?

**Comment by shbnm93** [comment]
same here.
```
RuntimeError: Unable to make 4067.4 MB space on cpu.
    GPU cache: 0 items; 0.0 MB; Max: 15487.0 MB;
    CPU cache: 0 items; 0.0 MB; Max: 0.0 MB;
    mem_free_total: 3571.3 MB; Ram Free: 5952.2 MB;
    ```

**Comment by brycedrennan** [comment]
What kind of system are you guys on?

**Comment by shbnm93** [comment]
> What kind of system are you guys on?

Windows

**Comment by nonoesp** [comment]
Hey @brycedrennan — Seeing the same error on a MacBook Pro M1 (2020) running macOS Sonoma.

**Comment by nonoesp** [comment]
As @aeroreyna mentioned in https://github.com/brycedrennan/imaginAIry/issues/347, you can get around this bug by downgrading to 12.0.3.

```sh
pip install imaginairy==12.0.3
```

**Comment by brycedrennan** [comment]
Yeah. basically I need to either:
1) undo the memory "management" i added
2) fix the memory management to work better
or
3) rewrite imaginairy entirely... which is the option I chose. High chance of failure here :-)

**Comment by brycedrennan** [comment]
Should work now in just released 14.0.0
